checker/default_checks.go

```diff
@@ -39,9 +39,9 @@ func getBackwardCompatibilityCheckConfig(checks []BackwardCompatibilityCheck, le
 }
 
 var optionalChecks = map[string]BackwardCompatibilityCheck{
-	"response-non-success-status-removed":   ResponseNonSuccessStatusRemoved,
-	"api-operation-id-removed":              APIOperationIdRemovedCheck,
-	"api-tag-removed":                       APITagRemovedCheck,
+	"response-non-success-status-removed":   ResponseNonSuccessStatusUpdated,
```

Comment:
We have a dilemma here:
Changing the check names would cause a breaking change for people who use the old names on the cmd-line.
But maintaining the current names (as you chose to do), could be misleading if the check produces errors that its name doesn't suggest.

Reply:
We don't need to resolve this now, just something we may want to improve moving forward

Reply:
Great point. I'd think second option is better as we don't affect users. Furthermore we are broadening checkers names for that reason, so that we can cover `added` and `removed` operations.